### ISSUE TITLE
Problem with minified HTML/CSS on IE7/IE8

### ISSUE THREAD

**Issue by igorprado** [keithclark/selectivizr#66]
I'm working with Middleman to test locally and build my files, minifying then.

Testing locally (where there are any kind of minifying), selectvizr works fine in IE7 and IE8. On build, where everything is minified, it doesn't work at all. And no error is given at console.

Solution for my case: disable HTML and CSS minify for instance, but I believe that's an issue.

Sorry my bad english.

**Comment by flatsteve** [comment]
I have the same issue. When I set Compass output_style = :compressed, Selectivizr does not append the .slvzr class.   

When I leave my stylesheet uncompressed Selectivizr works as expected.

If anyone has this same problem; setting output_style = :compact seems like the best compromise (when IE 8 support is essential to your project).

Thanks for a great script Keith.

**Comment by pluff** [comment]
+1 to this one. Script doesn't work with 1-lined css.

**Comment by CHEWX** [comment]
Yup +1, I have this problem too I think. It's the only reason why I can see why it is not working anyway.

**Comment by ArmorDarks** [comment]
Would be great to see fix for that... But since repo seems to be dead, there is very little hope for that...
